frontend/src/store/todos-context.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import TodosContextProvider, { TodosContext } from './todos-context';
+
+describe('TodoProvider', ()=>
+{
+	test('check length !== 0', ()=>
```

Comment:
Please use words only ... not code styles... what is the test doing? this does not tell me anything as long I read the whole test (what I dont want todo each time)